ibm/service/cis/resource_ibm_cis_filter.go

```diff
@@ -224,8 +224,7 @@ func ResourceIBMCISFilterValidator() *validate.ResourceValidator {
 			Identifier:                 cisFilterDescription,
 			ValidateFunctionIdentifier: validate.ValidateAllowedStringValue,
 			Type:                       validate.TypeString,
-			Required:                   true,
```

Comment:
@sdague,
To remove validation on description, just removing AllowedValues would not help.
Could you please remove the [validatefunc](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/d0c016927e35265c9dfb0a6ac80a7d5316b78072/ibm/service/cis/resource_ibm_cis_filter.go#L63) from the description schema and also [validator object](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/d0c016927e35265c9dfb0a6ac80a7d5316b78072/ibm/service/cis/resource_ibm_cis_filter.go#L223) for description from the validator function? Since the function has only one validator you can actually remove the [whole function](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/d0c016927e35265c9dfb0a6ac80a7d5316b78072/ibm/service/cis/resource_ibm_cis_filter.go#L220) and also its reference in provider.go file